udata/templates/territories/card.html

```diff
@@ -0,0 +1,47 @@
+{% cache cache_duration, 'territory-search-result', territory.id, g.lang_code %}
```

Comment:
A search on `territories/card.html` yields nothing, both in current code base and this PR. Is this used?

Reply:
https://github.com/dtc-innovation/udata/blob/093608c4a30d75b64941349a7114852c20a96e92/udata/templates/search.html

In the new search results apparently (:

Reply:
To be moved to another PR then ;-)